src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -155,9 +158,28 @@ CrashInfo::GatherCrashInfo(MINIDUMP_TYPE minidumpType)
         return false;
     }
 #endif
+    // Load and initialize DAC interfaces
+    if (!InitializeDAC())
+    {
+        return false;
+    }
+    // Enumerate all the managed modules. On MacOS only the native modules have been added
+    // to the module mapping list at this point and adds the managed modules. This needs to
+    // be done before the other mappings is initialized.
+    if (!EnumerateManagedModules())
+    {
+        return false;
+    }
+#ifdef __APPLE__
+    InitializeOtherMappings();
```

Comment:
What other mappings are there here? Other than native mappings and managed ones? Also, why is EnumerateMemoryRegions only called on macOS ?

Reply:
EnumerateMemoryRegions is the name of the MacOS function that does something similar to EnumerateModuleMappings() on Linux. I can change the name of one of them if that is important.